packages/elements/src/calendar/types.ts

```diff
@@ -39,3 +40,26 @@ export type WeekdayName = {
   narrow: string;
   long: string;
 };
+
+// public API
+export type CellModel = {
```

Comment:
I'm proposing `CalendarCell` instead. We tend to prefix public `types` or `functions` with element name to improve code readability and prevent clashes with other libs.

Reply:
Let's go with `CalendarCell`!